internal/pkg/template/templates/workloads/partials/cf/workload-container.yml

```diff
@@ -28,11 +28,13 @@
 {{- end}}
 {{- if eq .WorkloadType "Load Balanced Web Service"}}
   PortMappings:
-    - ContainerPort: !Ref ContainerPort
-    {{- if and .ServiceConnect .SCFeatureFlag}} # remove when release
+    {{- range $portMapping := .PortMappings}}
+    - ContainerPort: {{$portMapping.ContainerPort}}
+    {{- if and $.ServiceConnect $.SCFeatureFlag}} # remove when release
     {{/* Multiple exposed port is not supported yet. Therefore, if the target container is the sidecar, the target port must be the one and only one port that the container exposes. */}}
```

Comment:
Is this comment still useful?

Reply:
Yeah we can remove this comment, I will do it.